server/src/main/java/model/placement/fleet/Fleet.java

```diff
@@ -31,15 +32,15 @@ public Fleet() {
      * this method creates Fleet instance which consists of UndirectedShips
      * ship which is Undirected knows only about it's size but not about its
      * direction.
-     * @return
-     * Fleet which is sort of list which consists of UndirectedShips
+     *
+     * @return Fleet which is sort of list which consists of UndirectedShips
      */
     public static Fleet<UndirectedShip> createDefaultFleet() {
         Fleet<UndirectedShip> fleet = new Fleet<>();
-        ShipType[] shipTypes = {SHIP_MAST4,
-                SHIP_MAST3, SHIP_MAST3,
-                SHIP_MAST2, SHIP_MAST2, SHIP_MAST2,
-                SHIP_MAST1, SHIP_MAST1, SHIP_MAST1, SHIP_MAST1};
+        ShipType[] shipTypes = {FOUR_MAST,
+                THREE_MAST, THREE_MAST,
```

Comment:
Maybe method to create your own fleet (size and amount of ships)

Reply:
I thinks this falls into YAGNI category as modifiable fleet is an extra requirement :)